src/index.html

```diff
@@ -6,9 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo" >
+        <img class="image" src="images\logo.png"
+          alt="logo of moyo-header">
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__list" >
```

Comment:
As I can see in your styles, you are not using the `nav__list` class selector anywhere. You could get rid of that, it will make the HTML look cleaner ;)

P.S. if you were to use such a class for this `li` a better name for it would be something like `nav__item`